vars/Amazon.yml

```diff
@@ -1,4 +1,8 @@
 ---
+# The name of the S3 object corresponding to the GPG key for the
+# Nessus Agent system package
+package_gpg_key_object_name: RPM-GPG-KEY-Tenable-2048
+
```

Comment:
Shouldn't this file be deleted if we are not currently supporting any Amazon Linux distros?

Reply:
Removed in commit e5db5b2.